src/Adapter/Form/ChoiceProvider/TaxRuleGroupChoiceProvider.php

```diff
@@ -26,15 +26,28 @@
 
 namespace PrestaShop\PrestaShop\Adapter\Form\ChoiceProvider;
 
+use Address;
 use PrestaShop\PrestaShop\Core\Form\FormChoiceAttributeProviderInterface;
 use PrestaShop\PrestaShop\Core\Form\FormChoiceProviderInterface;
+use TaxManagerFactory;
 use TaxRulesGroup;
 
 /**
  * Provides tax rule group choices with tax rule name as key and id as value
  */
 final class TaxRuleGroupChoiceProvider implements FormChoiceProviderInterface, FormChoiceAttributeProviderInterface
 {
+    /** @var int */
+    private $countryId;
+    /** @var int */
```

Comment:
shouldn't cs fixer fail due to this? we usually use like this everywhere:
```suggestion
    
    /**
     * @var int
     */
```